src/main/resources/script.ddl

```diff
@@ -0,0 +1,13 @@
+create table json_data
```

Comment:
Only nullable data fields should have nullable columns in database. Not nullable data must be saved in not null columns.

Reply:
done